presto-mixin/dashboards/presto-overview.libsonnet

```diff
@@ -0,0 +1,962 @@
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 25,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'smooth',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'err/s',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local internalErrorFailuresOneMinuteRatePanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'com_facebook_presto_execution_QueryManager_InternalFailures_OneMinute_Rate{job=~"$job", presto_cluster=~"$presto_cluster"}',
+      datasource=promDatasource,
+      legendFormat='{{presto_cluster}}',
+      format='time_series',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Internal error failures - one minute rate',
+  description: 'The rate of internal failures occurring across the clusters.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisBorderShow: false,
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 25,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'smooth',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'err/s',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local alertsPanel = {
```

Comment:
Yea I can definitely move this up! Alerts have been added and verified (I can change the parameters to purposely make it fire for the sake of this mixin review if you'd like), the thing is the alerts only show when firing when you add a instance label filter. I asked vitaly about this during the last integration since the alerts show up even when normal/pending if you choose a datasource folder and no filter selector, but he told me that adding in the that instance label filter makes it so these arent used anymore 
<img width="705" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/38274348/d08b8f88-7f4f-4896-9cc5-31fd2ec35366">

Reply:
If you could set it to fire and add a screenshot here of it working that'd be grand! 😄

Reply:
<img width="844" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/38274348/1e0401a5-c292-4f2b-95bb-adfc409c25ac">